modules/user_accounts/php/NDB_Form_user_accounts.class.inc

```diff
@@ -300,80 +330,71 @@ class NDB_Form_User_Accounts extends NDB_Form
                         array('fullName' => $values['Real_name'])
                     );
                 } else {
+
                     $DB->update(
                         'examiners',
                         array(
-                         'radiologist' => $values['examiner_radiologist'] ?? 0,
+                         'radiologist' => $ex_radiologist,
                          'userID'      => $uid,
                         ),
                         array(
                          "full_name" => $values['Real_name'],
                         )
                     );
                     $examinerID = $examinerID[0]['examinerID'];
-                }
 
-                // Update CenterIDs
-                $ex_curr_sites = array();
-                $ex_prev_sites = array();
-
-                $prev_sites = $DB->pselect(
-                    "SELECT centerID
+                    //get existing sites for examiner
```

Comment:
section moved inside the else statement (line 302) since it's the only case where entries would need to be queried => avoid useless queries